nifi-nar-bundles/nifi-rules-action-handler-bundle/nifi-rules-action-handler-service/src/test/java/org/apache/nifi/rules/handlers/TestAlertHandler.java

```diff
@@ -129,11 +125,11 @@ public void testAlertWithBulletinLevel() {
         alertHandler.execute(reportingContext, action, metrics);
         BulletinRepository bulletinRepository = reportingContext.getBulletinRepository();
         List<Bulletin> bulletins = bulletinRepository.findBulletinsForController();
-        assertFalse(bulletins.isEmpty());
+        Assertions.assertFalse(bulletins.isEmpty());
         Bulletin bulletin = bulletins.get(0);
-        assertEquals(bulletin.getCategory(), category);
-        assertEquals(bulletin.getMessage(), expectedOutput);
-        assertEquals(bulletin.getLevel(), severity);
+        Assertions.assertEquals(bulletin.getCategory(), category);
+        Assertions.assertEquals(bulletin.getMessage(), expectedOutput);
+        Assertions.assertEquals(bulletin.getLevel(), severity);
```

Comment:
Is there a reason for switching from `import static` to the explicit `Assertions` reference? Replacing the import static references would reduce the number of changes significantly.